convert2rhel/systeminfo.py

```diff
@@ -15,20 +15,20 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import difflib
+import logging
+import os
+import re
+
 from collections import namedtuple
 
-from convert2rhel.utils import run_subprocess
+import six
 
+from convert2rhel.utils import run_subprocess
 
-try:
-    import ConfigParser as configparser
-except ImportError:
-    import configparser  # pylint: disable=import-error
 
-import difflib
-import logging
-import os
-import re
+six.add_move(six.MovedModule("configparser", "ConfigParser", "configparser"))
```

Comment:
I believe that the configparser move is built into six.  So this line is probably not necessary?